QuoteGeneration/quote_wrapper/common/inc/sgx_ql_lib_common.h

```diff
@@ -72,7 +72,7 @@ typedef enum _quote3_error_t {
     SGX_QL_UNABLE_TO_GENERATE_QE_REPORT = SGX_QL_MK_ERROR(0x0017),   ///< The QE was unable to generate its own report targeting the application enclave either
                                                                      ///< because the QE doesn't support this feature there is an enclave compatibility issue.
                                                                      ///< Please call again with the p_qe_report_info to NULL.
-    SGX_QL_KEY_CERTIFCATION_ERROR = SGX_QL_MK_ERROR(0x0018),         ///< Caused when the provider library returns an invalid TCB (too high).
+    SGX_QL_KEY_CERTIFICATION_ERROR = SGX_QL_MK_ERROR(0x0018),        ///< Caused when the provider library returns an invalid TCB (too high).
```

Comment:
Better to keep this typo until another major version. Both internal code and customer code use it. Changing the enum leads a build error.